docs/install.md

```diff
@@ -2,7 +2,9 @@
 
 :::{note} 
 
-AutoGluon requires Python version 3.8, 3.9, 3.10, or 3.11 and is available on Linux, MacOS, and Windows.
+* AutoGluon requires Python version 3.8, 3.9, 3.10, or 3.11 and is available on Linux, MacOS, and Windows.
+ 
+* The AutoGluon library comes pre-installed in all releases of [Amazon SageMaker Distribution](https://github.com/aws/sagemaker-distribution), the docker environment for data science used as the default image of [JupyterLab](https://docs.aws.amazon.com/sagemaker/latest/dg/studio-updated-jl.html) notebook instances in [Amazon SageMaker Studio](https://docs.aws.amazon.com/sagemaker/latest/dg/studio-updated.html).  SageMaker Studio notebooks users can access AutoGluon's automation capabilities without needing to install or configure anything additional. To find the AutoGluon and PyTorch versions available in a SageMaker Distribution image, refer to the [RELEASE.md](https://github.com/aws/sagemaker-distribution/blob/main/build_artifacts/v1/v1.4/v1.4.2/RELEASE.md) file for your image version in the SageMaker Distribution GitHub repository. 
```

Comment:
Another option is for `Amazon SageMaker` to be a new tab included in `Linux | Mac | Windows | Amazon SageMaker` in [the install page](https://auto.gluon.ai/dev/install.html). I'm ok with the current PR or this alternative.